scripts/helmcharts/openreplay/templates/job.yaml

```diff
@@ -151,6 +160,8 @@ spec:
           mountPath: /opt/openreplay
         - name: dbmigrationscript
           mountPath: /opt/migrations/
+      {{- end}}
+      {{- if .Values.kafka.enabled }}
```

Comment:
Same as clickhouse. We don't need this.

Reply:
Ah didn't realize how mandatory this was for the EE edition. Should be resolved now with https://github.com/openreplay/openreplay/commit/dd147fae9bdc428e542f9d85e9b8681c330b5c75